elastic-coders/serverless-webpack
# Serverless-offline and Serverless-webpack example

## What did you implement:
No related issue, this is a basic example to get folks started with the `serverless-offline` plugin. Although mentioned in both plugins documentation, there is no basic example and the lack of a flag `--location .webpack` caused a great number of issues for me which doesn't seem to be mentioned in either set of documentation but rather state "out of the box" support which was not the case.

Closes #XXXXX



## How did you implement it:
Duplication of existing babel example, upgrade of package dependencies to latest, integration of basic code to demonstrate usage.



## How can we verify it:

1. Clone directory and cd to `examples/serverless-offline`
2. Run `yarn`
3. Run `yarn run start`
4. Visit http://localhost:3000/first
5. Console should show successful response



## Todos:

- Fix linting errors (not yet implemented)
- Make sure code coverage hasn't dropped (not yet implemented)

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
